master/internal/logpattern/logpattern.go

```diff
@@ -61,9 +61,21 @@ func (l *logPatternPolicies) monitor(ctx context.Context,
 		if log.AgentID == nil {
 			return fmt.Errorf("agentID must be non nil to monitor logs")
 		}
+		// The first line of trial logs is printing expconf which has the regex pattern.
+		// We skip monitoring this line.
+		regex := "(.*)(\\\"log_pattern_policies\\\":)(.*)"
```

Comment:
can we make this a package level variable and use mustCompile since we will always need this regex 

Like we do for latin text
https://github.com/determined-ai/determined/blob/a33d2b8c862843a7f5ea3186d9f53500c83004ee/master/internal/api_user.go#L36